.github/CODEOWNERS

```
@@ -538,7 +538,7 @@
 /core/templates/Polyfills.ts @srijanreddy98
 /extensions/extensions.module.ts @vojtechjelinek
 /core/schema_utils*.py @oppia/data-and-stability-reviewers
-/core/utils*.py @aks681
+/core/utils*.py @oppia/lace-frontend-reviewers
```

```suggestion
/core/utils*.py @oppia/lace-backend-reviewers
```

Also, any specific reason for putting this utils file under LaCE team?

---

Not really. it's just many functions here are used by lace files and it was assigned to me before.